src/main/java/com/minecolonies/core/colony/Colony.java

```diff
@@ -119,14 +119,14 @@ public class Colony implements IColony
     private boolean ticketedChunksDirty = true;
 
     /**
-     * List of chunks that have to be be force loaded.
```

Comment:
no

Reply:
The doubled up "be" was even indicated by IntelliJ to be gramatically incorrect, so

Reply:
ohhh, I didn't even see it was a double "be", thought you removed the be completely. I think I should go back to bed =D